.circleci/scripts/run-deployment-tests.sh

```diff
@@ -68,6 +68,11 @@ ansible)
     docker run --rm \
         signalfx-agent-ansible-dev \
         ansible-lint .
+    if [ $CIRCLE_NODE_INDEX -eq 0 ]; then
```

Comment:
Does this only run it when changes are made to the playbook dir?

Reply:
Will only run for PRs if there are changes to the playbook dir or the ansible test dir (same as chef and puppet).